apps/emqx_authn/src/enhanced_authn/emqx_enhanced_authn_scram_mnesia.erl

```diff
@@ -131,14 +135,7 @@ authenticate(#{auth_method := AuthMethod,
 authenticate(_Credential, _State) ->
     ignore.
 
-destroy(#{user_group := UserGroup}) ->
-    trans(
-        fun() ->
-            MatchSpec = [{{user_info, {UserGroup, '_'}, '_', '_', '_', '_'}, [], ['$_']}],
-            ok = lists:foreach(fun(UserInfo) ->
-                                  mnesia:delete_object(?TAB, UserInfo, write)
-                               end, mnesia:select(?TAB, MatchSpec, write))
-        end).
+destroy(_St) -> ok.
```

Comment:
Q: intended to ignore this callback ?

Reply:
No, we are return it back too.
I should have drafted PR not to bother with updates before final review 😓